main.py

```diff
@@ -53,6 +64,16 @@ def setup_ui(self):
 
         self.ui.show()
 
+        # Create the loading gear but dont add it to anywhere, just save it
+        self.movie = QtGui.QMovie(self)
+        self.movie.setFileName("ui/res/loading.gif")
+        self.movie.setCacheMode(QtGui.QMovie.CacheAll)
+        self.movie.start()
```

Comment:
Will this play throughout the lifetime of the application or will it only play when needed?